engine/src/main/java/org/terasology/logic/characters/CharacterMoveInputEvent.java

```diff
@@ -36,11 +37,12 @@
     protected CharacterMoveInputEvent() {
     }
 
-    public CharacterMoveInputEvent(int sequence, float pitch, float yaw, Vector3f movementDirection, boolean running, boolean jumpRequested, long delta) {
+    public CharacterMoveInputEvent(int sequence, float pitch, float yaw, Vector3f movementDirection, boolean running, boolean crouching, boolean jumpRequested, long delta) {
```

Comment:
Agreeing with @flo that it would be nice to just add a new constructor and deprecate this one. Other modules than Pathfinding might also break.

Really you'd keep this change but then add a new deprecated constructor with the old signature + `@deprecated` that just calls the new one and nothing else.